### DISCUSSION TITLE
Need your help regarding incorrect colors 🙏

### DISCUSSION THREAD

**Discussion by danielshabat** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#496] [Q&A]
Hello everyone,
I have an ESP32, an SD module and 3 Led matrix, each of which is 64x32 pixels in size.
I try to activate them as shown in the example in the library - [AnimatedGIFPanel_SD](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/tree/master/examples/AnimatedGIFPanel_SD)
Everything works except that the colors that appear on the LED are not the right color, for example in the picture attached the caption should be yellow.
I will greatly appreciate your help.
![image](https://github.com/mrfaptastic/ESP32-HUB75-MatrixPanel-DMA/assets/36617419/5e12bb33-473f-44a5-95ef-1be30893ee96)

**Comment by board707** [comment]
Try to swap Green and Blue wires

**Reply by danielshabat** [reply]
Could you please tell me witch pin numbers related to the blue and green wires?

**Reply by board707** [reply]
G0 B0 and G1 B1 on the HUB75 connector

**Reply by danielshabat** [reply]
Thanks, it works 🙏

**Reply by mrcodetastic** [reply]
Thanks for your continued presence in this repository @board707 answering queries.

You continue to help many people. It's appreciated.
